sympy/stats/crv_types.py

```diff
@@ -1027,9 +1040,9 @@ class FDistributionDistribution(SingleContinuousDistribution):
     @staticmethod
     def check(d1, d2):
         _value_check(d1 > 0 and d1.is_integer, \
```

Comment:
`fuzzy_and` when `is_foo` is used unless `d1` cannot be symbolic (but if it can't then `d1.is_Integer` should be used.